src/infrastructure/NetworkService.js

```diff
@@ -0,0 +1,70 @@
+/*
+ *
+ * Copyright (c) 2019-present for NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License ");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import * as symbol from 'symbol-sdk'
+import Constants from '../config/constants'
+
+class NetworkService {
+  /**
+   * Get estimated effective rental fees for namespaces and mosaics
+   * @returns rental fees information
+   */
+  static getRentalFees = async () => {
+    const rentalFees = await new symbol.NetworkHttp('http://oleg-test-01.eu-west-1.symboldev.network:3000').getRentalFees().toPromise()
```

Comment:
yes, only for now, because only this endpoint is latest catapult-rest build.

https://nem2.slack.com/archives/GLK87TKM4/p1585812959003400

I will check on the other endpoint before merge this.